api/src/main/java/org/openmrs/parameter/OrderSearchCriteria.java

```diff
@@ -137,6 +190,14 @@ public OrderSearchCriteria(Patient patient, CareSetting careSetting, Collection<
 	 */
 	public Collection<OrderType> getOrderTypes() { return orderTypes; }
 
+	public String getAccessionNumber() {
```

Comment:
Would be useful to an add an @since annotation to any new public methods that your are introducing (here and elsewhere in this PR).

Reply:
Ah, agreed, will do!